app/src/host.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2021 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <kernel.h>
+#include <logging/log.h>
+
+#include <zmk/ble.h>
+#include <zmk/endpoints.h>
+#include <zmk/hid.h>
+#include <zmk/host.h>
+#include <zmk/led_indicators.h>
+
+LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
+
+#ifdef CONFIG_ZMK_USB
+
+#define LED_REPORT_ID 1
+
+void zmk_host_process_usb_report(struct zmk_host_usb_report report) {
```

Comment:
Why is this here, and not in `app/src/usb.c`?

Seems like USB specifics leaking in here.